workflow/Snakefile

```diff
@@ -168,7 +168,8 @@ rule run_predictions:
 		threshold = config['params_predict']['threshold'],
 		flags = config['params_predict']['flags'],
 		gamma = config['params_predict']['hic_gamma'],
-		scale = config['params_predict']['hic_scale']
+		scale = config['params_predict']['hic_scale'],
+		default = lambda wildcards: biosamples_config.loc[wildcards.biosample, 'default_accessibility_feature']
```

Comment:
Can you use the `get_accessibility_file` function? Similar to the other snakemake rules

Reply:
This is to get the value of the "default_accessibility_feature" in the input bio sample table. I can add a `get_accessibility_feature` function.

Reply:
You don't need to add the function. It already exists!